src/Control/Parallel/Class.purs

```diff
@@ -0,0 +1,34 @@
+module Control.Monad.Parallel.Class 
+  ( MonadPar, par
+  , MonadRace, race
+  , Parallel(), inParallel, runParallel
+  ) where
+
+import Prelude
+import Control.Alt
+
+class (Monad m) <= MonadPar m where
+  par :: forall a b. m (a -> b) -> m a -> m b
+
+class (MonadPar m) <= MonadRace m where
+  race :: forall a. m a -> m a -> m a
+
+newtype Parallel m a = Parallel (m a)
```

Comment:
This is good, factors out the newtype that a lot of libs will do (e.g. `Aff` and `Par`).